manifests/07-downloads-deployment.yaml

```diff
@@ -1,177 +0,0 @@
-apiVersion: apps/v1
```

Comment:
No we dont need to remove them since they are static.
If it's ok, I would rather keep it simple and create additional card for it, since the the PR is pretty big even now.